jwql/website/apps/jwql/data_containers.py

```diff
@@ -69,6 +70,51 @@
 REPO_DIR = os.path.split(PACKAGE_DIR)[0]
 
 
+def build_table(tablename):
+    """Create Pandas dataframe from JWQLDB table.
+
+    Parameters
+    ----------
+    tablename : str
+        Name of JWQL database table name.
+
+    Returns
+    -------
+    table_meta_data : pandas.DataFrame
+        Pandas data frame version of JWQL database table.
+    """
+    # Make dictionary of tablename : class object
+    # This matches what the user selects in the select element
+    # in the webform to the python object on the backend.
+    tables_of_interest = {}
+    for item in di.__dict__.keys():
+        table = getattr(di, item)
+        if hasattr(table, '__tablename__'):
+            tables_of_interest[table.__tablename__] = table
+
+    session, _, _, _ = load_connection(get_config()['connection_string'])
+    table_object = tables_of_interest[tablename]  # Select table object
```

Comment:
I get a KeyError here when selecting `nirspec_bias_stats` from the dropdown. I think this is just because this table doesn't exist yet, but I wasn't sure if the dropdown options are supposed to account for that or if this is expected.

Reply:
Hey Ben, that is the case with some of the tables. I am also using the `dev` database as well.

Reply:
Were you able to resolve this using the `dev` database tables? I was hoping we could merge soon.

Reply:
No, it still gives that error with the dev database. Though I think this is just because these aren't officially incorporated into the dev database yet (i.e. I don't see them listed in `database_interface.py`), so I think it will be fixed in the future once they are.

Reply:
Yeah, its not really a surprise that there are inconsistencies amongst dev/test/ops database.  They are all in different states.  Hopefully things will get better once we have #585 and a better dev/test/ops workflow.